AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Commands/PrepareRelease.swift

```diff
@@ -0,0 +1,119 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import ArgumentParser
+import Foundation
+import PackageDescription
+
+struct PrepareRelease: ParsableCommand {
```

Comment:
Don't love this name. Need to think on it more, but this is good enough for now. Please lmk if you have better suggestions.